notebooks/pregnancy/maternal_brain_project/r2_pvalues_all_enigma_meshes_new.ipynb

```diff
@@ -0,0 +1,1445 @@
+{
```

Comment:
<p>pca = EvaluatedModel(..) --&gt; Super clear! Thanks.</p><p>NIT: I don't know if we'd say "reconstruction" for the week --&gt; pcs parts. Maybe replace with "Prediction"?</p><p> </p><p>Excellent, all of this is super clear.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geometric-intelligence/herbrain/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>